clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/loadbalancer/view/AzureLoadBalancerProvider.groovy

```diff
@@ -78,6 +78,7 @@ class AzureLoadBalancerProvider implements LoadBalancerProvider<AzureLoadBalance
       name: loadBalancerDescription.loadBalancerName,
       region: loadBalancerDescription.region,
       vnet: loadBalancerDescription.vnet?: "vnet-unassigned",
+      subnet: loadBalancerDescription.subnet?: "subnet-unassigned",
```

Comment:
Might read easier with a space before the elvis operator. Can always change this in a subsequent PR.